src/Doctrine/Odm/Filter/ManagerRegistryConfigurableInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Doctrine\Odm\Filter;
+
+use Doctrine\Bundle\MongoDBBundle\ManagerRegistry;
+
+interface ManagerRegistryConfigurableInterface
+{
+    public function setManagerRegistry(?ManagerRegistry $managerRegistry): ?ManagerRegistry;
```

Comment:
```suggestion
    public function setManagerRegistry(ManagerRegistry $managerRegistry): void;
```

Can you add `getManagerRegistry` and `hasManagerRegistry` to the interface?